csharp/Platform.Collections/IDictionaryExtensions.cs

```diff
@@ -4,17 +4,83 @@
 
 #pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
 
-namespace Platform.Collections
+namespace Platform.Collections 
 {
+    /// <summary>
+    /// <para>Represents a set of extension methods <see cref="IDictionary{TKey, TValue}"/> array.</para>
+    /// <para>Представляет набор методов расширения <see cref="IDictionary{TKey, TValue}"/>.</para> 
+    /// </summary>  
     public static class IDictionaryExtensions
     {
+        /// <summary>    
+        /// <para> 
+        /// Returns the value associated with the passed parameters  
+        /// <paramref name="key"/> in <paramref name="dictionary"/> 
+        /// </para>  
+        /// <para> 
+        /// Возвращает значение связанное с  
+        /// <paramref name="key"/> в  
+        /// <paramref name="dictionary"/> или значение по умолчанию, если оно не существует.
+        /// </para> 
+        /// </summary>   
+        /// <paramref name="dictionary"> 
```

Comment:
I did not mean `use paramref instead of typeparam`. I wanted to see paramrefs in the sentences instead of plain words.
Example: `associated with the <paramref name="key"/>` is correct. `associated with the key` is incorrect.